### PR TITLE
Add quick start app for web

### PR DESCRIPTION
**Pull request by imraymondlee**
Adding the quick start app for web built from [this page](https://docs.aws.amazon.com/location/latest/developerguide/getting-started.html) to allow users to download the completed app.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by dreamorosi — APPROVED** [review]
I would suggest you to consider using AWS SDK v3 and include only the Location client instead of the whole AWS SDK. Given we are on web reducing bundle size would help with page speed, even though this is a demo.

Aside from this you have the approval, if you don't plan on swapping the SDK client let me know with a comment below and I'll merge right away.

**Comment by imraymondlee** [comment]
We won't be upgrading the v3 at the moment as v3 requires a new process for the user to set up the app.

**Pull request merged**
